internal/ghrepo/repo.go

```diff
@@ -18,7 +18,7 @@ type Interface interface {
 
 // New instantiates a GitHub repository from owner and name arguments
 func New(owner, repo string) Interface {
-	return NewWithHost(owner, repo, ghinstance.OverridableDefault())
```

Comment:
This function is being used in `repo create`, so changing its behavior from respecting GH_HOST to not respecting it anymore may change `repo create` behavior. Perhaps `repo create` should be using `NewWithHost`?

Reply:
I believe I changed the usage in `repo create` to now use `ghrepo.NewWithHost`. Was there another spot that I missed? I did a quick search and I did not find any usage of this function outside of tests.